modules/odo-core-concepts.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/developer_cli_odo/understanding-odo.adoc
+
+:_content-type: CONCEPT
+[id="odo-core-concepts_{context}"]
+
+= odo core concepts
+
+`{odo-title}` abstracts Kubernetes concepts into a developer friendly terminology:
```

Comment:
In your preview, does the `{odo-title}` attribute render the way you expect it to?

Reply:
Would it help to give the full name of odo instead: Red Hat® OpenShift® Do ( odo )

Reply:
```suggestion
`odo` abstracts Kubernetes concepts into a developer friendly terminology:
```